.github/workflows/check.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           node-version: lts/jod
       - run: npm ci
       - run: npm run lint
+      - run: npm run prettier:check
```

Comment:
```suggestion
      - run: node --run lint
      - run: node --run prettier:check
```

Reply:
I plan to do a separate PR to update `npm run` across the repo 👍 Especially when calling other scripts within scripts.